src/components/styledComponents.jsx

```diff
@@ -0,0 +1,35 @@
+import styled, { createGlobalStyle } from "styled-components";
+import reset from 'styled-reset';
+
+const GlobalStyle = createGlobalStyle`
+  ${reset};
+`;
+
+const HourCell = styled.div`
+  position: relative;
+  display: inline-block;
+  height: 10vh;
+  width: 15wh;
+  justify-content: start;
+`
+
+const RoomCell = styled.div`
+  position: relative;
+  display: inline-block;
+  height: 10vh;
+  width: 15wh;
+  justify-content: center;
+`
+
+const BasicCell = styled.div`
+  position: relative;
+  display: block;
+  height: 4vh;
+  width: 6wh;
+  text-align: ${props => props.alignCell};
+  background: ${props => props.backgroundColor};
+`
```

Comment:
I'll have to look up exactly how this syntax works cause I've forgotten, but I assume this is just the proper syntax for 'determine this style by looking at the props.'